StreamChatClient_v3/Config/ChatClientConfig.swift

```diff
@@ -47,3 +47,25 @@ extension ChatClientConfig {
         // something
     }
 }
+
+/// The API key of the app. You can obtain this value ... TODO
+public struct APIKey: Equatable {
+    /// The string representation of the API key
+    public let apiKeyString: String
+    
+    /// Creates a new `APIKey` from the provided string. Fails, if the string is empty.
+    public init?(_ apiKeyString: String) {
+        guard apiKeyString.isEmpty == false else {
+            log.error("Can't initialize APIKey with an empty string.")
+            return nil
+        }
+        self.apiKeyString = apiKeyString
+    }
+    
+    /// Creates a new `APIKey` from the provided static string.
+    public init(_ apiKeyString: StaticString) {
+        let string = "\(apiKeyString)"
+        assert(string.isEmpty == false, "APIKey can't be initialize with an empty string.")
```

Comment:
We can use `log.assert`

Reply:
fixed in 6793e3301810e0d7658d51aa01ac85db5b35ac87